bookwyrm/templates/shelf/search_filter_field.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'snippets/filters_panel/filter_field.html' %}
+{% load i18n %}
+
+{% block filter %}
+  <div class="control">
+    <label class="label" for="search_query">{% trans 'Search by keyword' %}</label>
```

Comment:
"search by keyword" is a bit confusing when this is a "filter" box. Technically we're doing both, but I wonder if it would be clearer to make this "filter by keyword" or "enter keywords to filter your books" or something similar.